examples/iam-policy/main.tf

```diff
@@ -53,3 +53,28 @@ module "iam_policy_from_data_source" {
     PolicyDescription = "Policy created using example from data source"
   }
 }
+
+module "iam_policy_optional" {
+  source        = "../../modules/iam-policy"
+  create_policy = false
+
+  name        = "example_optional"
```

Comment:
There are `default` values already specified for all variables in the module, so you can remove everything starting from line 61 ([like here](https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/examples/complete-vpc/main.tf#L171-L175)).

Reply:
Ok, I've removed "path" and "description".

Thank you for your time!

Reply:
Please remove everything and leave just `source` and `create_policy = false` with empty line in between.

Reply:
Done.